kore/src/Kore/Builtin/Set.hs

```diff
@@ -372,6 +376,23 @@ evalIntersection =
             _set2 <- expectConcreteBuiltinSet ctx _set2
             returnConcreteSet resultSort (Map.intersection _set1 _set2)
 
+evalList2set :: Builtin.Function
+evalList2set =
+    Builtin.functionEvaluator evalList2set0
+  where
+    evalList2set0 :: Builtin.FunctionImplementation
+    evalList2set0 _ resultSort arguments =
+        Builtin.getAttemptedAxiom $ do
+            let _list =
+                    case arguments of
+                        [_map] -> _map
+                        _      -> Builtin.wrongArity Set.list2setKey
+            _list <- List.expectConcreteBuiltinList Set.list2setKey _list
+            let _set = Map.fromList
+                           $ fmap (\x -> (x, Domain.SetValue))
```

Comment:
This doesn't look like a multiple of 4 white spaces here. Also, the line below can be aligned just below this one.